src/background_remover.cc

```diff
@@ -268,8 +268,38 @@ static cv::Mat resizeAndPadTo(const cv::Mat &frame, int targetw, int targeth, Pa
     return ret;
 }
 
-void BackgroundRemover::maskBackground(cv::Mat &frame /* rgb */, const cv::Mat &maskImage /* rgb */,
-                                       bool do_blur_mask, Timing &t) {
+void BackgroundRemover::blendPixels(const cv::Vec3b &a,
+                                    const cv::Vec3b &b,
+                                    uint8_t p,
+                                    cv::Vec3b &result) {
+    for (int c = 0; c < 3; c++) {
+        int mix = 0;
+        mix += (p * b[c]) / 255;
+        mix += ((255-p) * a[c]) / 255;
+        result[c] = mix;
+    }
+}
+
+void BackgroundRemover::blendLayers(const cv::Mat &foreground,
+                                    const cv::Mat &background,
+                                    const cv::Mat &mask,
+                                    cv::Mat &output) {
+    for (int x = 0; x < foreground.cols; x++) {
+        for (int y = 0; y < foreground.rows; y++) {
+            cv::Point p(x, y);
+            uint8_t m = mask.at<uint8_t>(p);
```

Comment:
Please use `unsigned char` for consistency like in the caller.

Reply:
Fine, but it's sooooooo long :-D